### PR TITLE
Gazebo packages

### PR DESCRIPTION
**Pull request by edith-langer**
It seems that on a computer with only ros-indigo-desktop (and not ros-indigo-desktop-full) installed, two packages concerning gazebo have to be installed to be able to use our simulation.

I launched the robotino_bringup_sim and got an unconnected tf-tree (no connection between odom and base_link). Installing the two packages fixed this issue!

### CODE REVIEW

File: robotino_controller_configuration_gazebo/package.xml [hunk 1]
```diff
@@ -13,6 +13,8 @@
   <build_depend>roslaunch</build_depend>
   <build_depend>rostopic</build_depend>
   <build_depend>gazebo_ros_control</build_depend>
+  <build_depend>ros-indigo-gazebo-plugins </build_depend>
```
Comment (NadiaHG) [inline robotino_controller_configuration_gazebo/package.xml]: does it work? I though that  to the package.xml you can only add the ros name of the packages and not the ubuntu distro name. 

and where are used on squirrel_controller_configuration_gazebo the gazebo_plugins package?

File: robotino_controller_configuration_gazebo/package.xml [hunk 2]
```diff
@@ -13,6 +13,8 @@
...
+  <build_depend>ros-indigo-gazebo-ros-pkgs</build_depend>
```
Comment (NadiaHG) [inline robotino_controller_configuration_gazebo/package.xml]: gazebo_ros_pkgs is a metapackage , please dont add metapackages as dependency

**Review by ipa-nhg — CHANGES_REQUESTED** [review]

**Comment by edith-langer** [comment]
You are of course right about the metapackage. I removed it from package xml.
And I changed ros-indigo-gazebo-plugins to gazebo-plugins. I tested it with that configuration and it works now.

I definitely need the gazebo-plugins otherwise I get errors when I want to start the simulation. 

```
Error [Plugin.hh:156] Failed to load plugin libgazebo_ros_bumper.so: libgazebo_ros_bumper.so: cannot open shared object file: No such file or directory
Error [Plugin.hh:156] Failed to load plugin libgazebo_ros_laser.so: libgazebo_ros_laser.so: cannot open shared object file: No such file or directory
```

I don't know in which package it would fit better?

**Comment by ipa-nhg** [comment]
Hi @edith-langer you should add the dependency to robotino_description https://github.com/squirrel-project/squirrel_common/blob/indigo_dev/robotino_description/package.xml I though that gazebo_ros included the plugins , but you are right , it is not included. could you please add the dependency to robotino_description? 

Thanks.

**Comment by ipa-nhg** [comment]
can be closed  https://github.com/squirrel-project/squirrel_common/commit/71af83b1d5103fb3ad1c3ec5684e36bdc47740cd

thanks @edith-langer

**Pull request closed without merging**